omx/gstomxaudiodec.c

```diff
@@ -345,19 +337,19 @@ gst_omx_audio_dec_loop (GstOMXAudioDec * self)
     /* Just update caps */
     GST_AUDIO_DECODER_STREAM_LOCK (self);
 
-    gst_omx_port_get_port_definition (port, &port_def);
-    g_assert (port_def.format.audio.eEncoding == OMX_AUDIO_CodingPCM);
```

Comment:
The port_def variable is unused with this change, so please remove it from line 309.

Reply:
66cee90a7d9904dfc8155591da324492d4f520ef